front-end/src/components/PostDetailModal.tsx

```diff
@@ -57,6 +62,10 @@ export default function PostDetailModal(props:Props) {
       <ModalBody style={{wordWrap: 'break-word'}}>
         <h6 className="mb-2 text-muted">By: {post.author.displayName}</h6>
         <span className="mb-2 text-muted">{getDateString(post)}</span>
+        <div style={{height: "1em"}} >
+          {/* I'm being hypocritical here using CardLink in a Modal */}
+          <CardLink onClick={()=>copyLinkToClipboard(post)}>{linkIcon}</CardLink>
+        </div>
```

Comment:
Some padding on the bottom might be nice since in the screenshot it seems to be a little close to the text. Or maybe it being on the top right corner might be nice? Up to you but looks good!!

Reply:
Good call! Implemented some bottom padding